src/store/channels-list/saga.ts

```diff
@@ -139,6 +158,13 @@ export function* fetchConversations() {
     chatClient,
     chatClient.getConversations,
   ]);
+
+  const matrixIds = extractMatrixIdsFromConversations(conversations);
+  if (matrixIds.length > 0) {
+    yield call(fetchMissingUsersData, matrixIds);
+  }
+
+  yield call(updateMessageSenders, conversations);
```

Comment:
We already have a `mapToZeroUsers` method. Seems like it makes sense to do this there rather than doing this mapping in 2 separate steps.

But, thinking more high level, the issue here is that we usually base our fetches on the `otherMembers` and some members may not be in the conversation any more, right? Maybe we just need a new field on the conversation that holds everyone who was ever invited to a conversation? Then we wouldn't need to sift through every single message.